62_CAD/common.hlsl

```diff
@@ -41,14 +42,14 @@ struct DrawObject
 
 struct LinePointInfo
 {
-    float64_t2 p;
+    nbl::hlsl::portable_vector64_t2 p;
     float32_t phaseShift;
     float32_t stretchValue;
 };
 
 struct QuadraticBezierInfo
 {
-    nbl::hlsl::shapes::QuadraticBezier<float64_t> shape; // 48bytes = 3 (control points) x 16 (float64_t2)
+    nbl::hlsl::shapes::QuadraticBezier<nbl::hlsl::portable_float64_t<> > shape; // 48bytes = 3 (control points) x 16 (emulated_float64_t)
```

Comment:
what's with the extra <>, template stuff compile error?

Reply:
room for device capability traits (the template param gets defaulted to `void`), the JIT device traits should be passed here once all the emulated float64 testing is complete

Reply:
i already do emulated_float64_t<nbl::hlsl::jit::device_capabilities>

Reply:
do not introduce a dependency on `jit` namespace outside the examples

Reply:
so far only vertex shader of example 62 is dependent on jit

Reply:
@Erfan-Ahmadi resolve when read